.codespellrc

```diff
@@ -1,7 +1,7 @@
 # See: https://github.com/codespell-project/codespell#using-a-config-file
 [codespell]
 # In the event of a false positive, add the problematic word, in all lowercase, to a comma-separated list here:
-ignore-words-list = ,
+ignore-words-list = WAN,
```

Comment:
```suggestion
ignore-words-list = wan,
```
Codespell itself is case insensitive (so "WAN" matches the "wan" word in its misspelling dictionary), but the ignore words capability is case sensitive. So the word added to the list must match exactly with the case of the word in Codespell's misspelling dictionary. Since that dictionary is all lower case, the ignore word list must also be all lower case.

Reply:
Thanks for that explanation and suggestion @per1234 , that makes sense!